src/test/integration/collections.e2e-spec.ts

```diff
@@ -71,11 +82,40 @@ describe('Collection Service', () => {
     });
   });
 
+
   describe('Collections count', () => {
     it(`should return a number`, async () => {
       const nftCount: Number = new Number(await collectionService.getNftCollectionCount(new CollectionFilter()));
 
       expect(nftCount).toBeInstanceOf(Number);
     });
   });
+
+  describe('Get Collection for a specific address', () => {
```

Comment:
I think a constant account address should be set for these tests (accountAddress could not be always the same)